EU4toV2/Source/V2World/V2World.cpp

```diff
@@ -615,15 +616,16 @@ bool scoresSorter(pair<V2Country*, int> first, pair<V2Country*, int> second)
 void V2World::convertNationalValues(const mappers::IdeaEffectMapper& ideaEffectMapper)
 {
 	// set national values
-	list< pair<V2Country*, int> > libertyScores;
-	list< pair<V2Country*, int> > equalityScores;
+	list< pair<V2Country*, double> > libertyScores;

```

Comment:
Can compress the << and >> now:
list<pair<V2Country*, double>>

Reply:
compressed.